src/pages/MyNotes/components/FilterMenu/FilterMenuContainer.tsx

```diff
@@ -0,0 +1,58 @@
+import { EMPTY_STRING, FILTER_OPTIONS } from 'config';
+import { useToggle } from 'hooks';
+import { setFilter } from 'store/features';
+import { useAppDispatch, useAppSelector } from 'store/hooks';
+import { getMyNotes } from 'store/selectors';
+import { formatDate } from 'utils';
+
+import FilterMenu from './FilterMenu';
+import { IFilterByDateValues, IFilterByTitleValues } from './types';
+
+const FilterMenuContainer = () => {
+  const { isFilteredByDate, isFilteredByTitle } = useAppSelector(getMyNotes);
+  const [isFilterByTitleOpen, setFilterByTitleOpen] = useToggle(false);
+  const [isFilterByDateOpen, setFilterByDateOpen] = useToggle(false);
+  const dispatch = useAppDispatch();
+
+  const FilterByTitleInitialValues: IFilterByTitleValues = {
+    titleForFilter: EMPTY_STRING,
+  };
+  const FilterByDateInitialValues: IFilterByDateValues = {
+    dateForFilter: undefined,
+  };
```

Comment:
should be in constants.ts and in uppercase
also dateForFilter should be null, you can't set undefined